File: 1365-how-many-numbers-are-smaller-than-the-current-number/1365-how-many-numbers-are-smaller-than-the-current-number.go
func smallerNumbersThanCurrent(nums []int) []int {
    n, ln := 0, len(nums)
    var smaller []int
    
    for i := 0; i < ln; i++ {
        n = 0
        for j := 0; j < ln; j++ {
            if i != j && nums[j] < nums[i] {
                n++
            }
        }
        smaller = append(smaller, n)
    }
    
    return smaller
}

File: 1365-how-many-numbers-are-smaller-than-the-current-number/1365-how-many-numbers-are-smaller-than-the-current-number.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        n, ln = 0, len(nums)
        smaller = []
        for i in range(ln):
            n = 0
            for j in range(ln):
                if i != j and nums[j] < nums[i]:
                    n += 1
            smaller.append(n)
        
        return smaller
                    
            